netstandard/ref/System.Reflection.Emit.cs

```diff
@@ -28,10 +36,13 @@ public sealed partial class AssemblyBuilder : System.Reflection.Assembly
 //      public void Save(string assemblyFileName, System.Reflection.PortableExecutableKinds portableExecutableKind, System.Reflection.ImageFileMachine imageFileMachine) { }
         public void SetCustomAttribute(System.Reflection.ConstructorInfo con, byte[] binaryAttribute) { }
         public void SetCustomAttribute(System.Reflection.Emit.CustomAttributeBuilder customBuilder) { }
+        public void SetEntryPoint(System.Reflection.MethodInfo entryMethod) { }
+        public void SetEntryPoint(System.Reflection.MethodInfo entryMethod, System.Reflection.Emit.PEFileKinds fileKind) { }
     }
     [System.FlagsAttribute]
     public enum AssemblyBuilderAccess
     {
+        ReflectionOnly = 6,
```

Comment:
We did expose ReflectionOnlyLoad methods in .NET Standard. Any reason to exclude this enum member?

Reply:
Same as above. ReflectionOnlyLoad throws PNSE in .NET Core.

Reply:
I see, in that case let's not add the enum.